setup.py

```diff
@@ -21,7 +21,9 @@
 
 VERSION = open('VERSION', 'r').read().strip()
 
-VIRTUAL_ENV = hasattr(sys, 'real_prefix')
```

Comment:
I believe it was named like that because it was a constant and the pylint warns to use upper case.

Anyway, renaming and making it a private method vanish the confusion I guess

Reply:
A bit too late for review, but it's exactly how @wainersm said. Global name variables must be UPPER_CASE. Anyway I'd prefer it to stay as a variable, because with this patch every `get_dir` needs to call additional function that assess whether we are in virtual env or not, which is not likely to change throughout the execution, is it?

Reply:
(I mean should, not must, but ...)

Reply:
@ldoktor it never was about the variable being upper case, pylint of course warned me when I messed with that, the confusing part to me was the naming which was IMO wrong. I don't think it really matters from performance's perspective whether it's a variable or a oneliner method.